src/Markdown.MAML/Model/MAML/MamlCommand.cs

```diff
@@ -31,6 +31,8 @@ public List<MamlParameter> Parameters
 
         public bool SupportCommonParameters { get; set; }
 
+        public string ModuleName { get; set; }
```

Comment:
From looking at the code where we read this property, null is an acceptable value.